gateway/src/kafka.rs

```diff
@@ -139,8 +147,13 @@ fn start_consumer(
                 };
 
                 consumer.commit_message(&msg, CommitMode::Async).unwrap();
-                if let Ok(move_made) = serde_json::from_str(payload) {
-                    events_in.send(Events::MoveMade(move_made)).unwrap()
+                match serde_json::from_str(payload) {
+                    Ok(Events::MoveMade(m)) => events_in.send(Events::MoveMade(m)).unwrap(),
+                    Ok(Events::HistoryProvided(h)) => {
+                        events_in.send(Events::HistoryProvided(h)).unwrap()
+                    }
+                    Ok(_) => (),
```

Comment:
This isn't necessary.  We should split the kafka-bound events from the non-kafka-bound events using the type system